scripts/ci-e2e.sh

```diff
@@ -55,6 +55,7 @@ export TAG="${defaultTag:-dev}"
 export GINKGO_NODES=3
 
 export AZURE_LOCATION="${AZURE_LOCATION:-$(capz::util::get_random_region)}"
+export AZURE_LOCATION_GPU="${AZURE_LOCATION_GPU:-$(capz::util::get_random_region_gpu)}"
```

Comment:
there is also ci-conformance.sh, don't think we currently run conformance on GPU clusters but it's worth adding it so it's consistent / in case we do in the future